### PR TITLE
Add support to fio to collect only the info requested, fix indentation

### LINKED ISSUE #599: Pbench-fio should have the ability to collect only the info requested
By default collect-sysinfo collects kernel_config, block, libvirt, sos, block and ara if installed and configured. It should allow users to choose whether to collect sysinfo or not. If yes, it should support 
passing the info to be collected as comma separated values when starting a benchmark run.

### PR DESCRIPTION
**Pull request by chaitanyaenr**
Fixes #599 

- By default collect-sysinfo collects kernel_config, block, libvirt, sos,
block and ara if installed and configured. This commit will allow users
to choose whether to collect sysinfo or not. This also adds ability to
pass the info to be collected as comma separated values when starting
a benchmark run.

- Fix indentation.

### CODE REVIEW

**Comment by chaitanyaenr** [comment]
Needs unit tests.

File: agent/bench-scripts/pbench-fio [hunk 1]
```diff
@@ -742,6 +759,6 @@ pbench-metadata-log --group=$tool_group --dir=$benchmark_run_dir end
 echo "$script_path/postprocess/generate-benchmark-summary \"$benchmark\" \"$orig_cmd\" \"$benchmark_run_dir\"" >"$benchmark_run_dir/generate-benchmark-summary.cmd"
 chmod +x "$benchmark_run_dir/generate-benchmark-summary.cmd"
 $script_path/postprocess/generate-benchmark-summary "$benchmark" "$orig_cmd" "$benchmark_run_dir"
-pbench-collect-sysinfo --group=$tool_group --dir=$benchmark_run_dir end
+pbench-collect-sysinfo --group=$tool_group --dir=$benchmark_run_dir  --sysinfo=$sysinfo end
```
Comment (ndokos) [inline agent/bench-scripts/pbench-fio]: You need to specify --sysinfo on the *other* pbench-collect-sysinfo call as well: the one at the beginning of the run.

Comment (chaitanyaenr) [inline agent/bench-scripts/pbench-fio]: Fixed.

**Comment by chaitanyaenr** [comment]
@ndokos added unit tests, please take a look.

**Pull request merged**